tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-minversion = 1.4.2
+minversion = 3.4.0
```

Comment:
So, I am trying to understand why a newer version of tox is needed?

Reply:
To support this --> 
 `ANSIBLE_INVENTORY_FILE {env:ANSIBLE_INVENTORY_FILE:/etc/ansible/hosts}`